.github/workflows/validate.yml

```diff
@@ -10,4 +10,6 @@ on:
 
 jobs:
   validate:
-    uses: ronin-co/actions/.github/workflows/validate.yml@main
\ No newline at end of file
+    uses: ronin-co/actions/.github/workflows/validate.yml@main
+    with:
+      package_dir: './packages/blade'
```

Comment:
Not a blocker, but would be great if this would run at the root so that the CI runs for all packages.